openssl/src/crypto/rsa.rs

```diff
@@ -42,14 +42,14 @@ impl Drop for RSA {
 impl RSA {
     /// only useful for associating the key material directly with the key, it's safer to use
     /// the supplied load and save methods for DER formatted keys.
-    pub fn from_public_components(n: BigNum, e: BigNum) -> Result<RSA, ErrorStack> {
+    pub fn from_public_components(n: BigNum,
```

Comment:
Wait, `d` is the private exponent. I don't think it makes sense to pass it in here.